src/Microsoft.Framework.Runtime.Hosting/project.json

```diff
@@ -4,6 +4,7 @@
     "dependencies": {
         "Newtonsoft.Json": "6.0.6",
         "NuGet.ProjectModel": "3.0.0-*",
+        "NuGet.RuntimeModel": "3.0.0-*",
```

Comment:
Ah, but Runtime.Hosting doesn't need a direct dependency of course... duh. Anyway, it does still need to be internalized.